assemblies/assembly-planner-configuration.adoc

```diff
@@ -378,6 +369,7 @@ In Eclipse, `debug` logging to the console tends to cause congestion with score
 Set the logging level to `debug` logging to see when the phases end and how fast steps are taken.
 
 The following example shows output from debug logging:
+COMMENT: Anna, we problem need to copy real log data.
 
 [source,options="nowrap"]
 ----
```

Comment:
"Configure the logging level on the org.optaplanner package in your logback.xml file as shown in the following example where <LEVEL> is a logging level listed in Section 9.4, “Using Logback to log OptaPlanner solver activity”. "
This is a chapter that reffers to itself, I think you need to change it to reffer to 9.3 instead of 9.4

The log level command in xml looks

` <logger name="org.optaplanner" level="${logback.level.org.optaplanner:-debug}"/>`
So beside user need to add a flag with log level he needs to use `logback.level.org.optaplanner` as well